requirements.txt

```diff
@@ -9,5 +9,3 @@ click==6.7
 Babel==2.6.0
 Flask-Caching==1.4.0
 pymongo==3.7.0
-azure-keyvault=1.1.0
```

Comment:
You'll need to have it in both requirements.txt and setup.py for it to pass CI, that's what the "Check dependency lists are synced" stage is looking for, that the lists are synced